### PR TITLE
re-added missing buttons

### PR DESCRIPTION
**Pull request by techniic0l0r**

### CODE REVIEW

File: src/components/layout/SiteLayout.tsx [hunk 1]
```diff
@@ -1,6 +1,9 @@
 import React from "react";
 import Link from "next/link";
 import Head from "next/head";
+import { faEnvelope } from "@fortawesome/free-solid-svg-icons/faEnvelope";
```
Comment (travigd) [inline src/components/layout/SiteLayout.tsx:4]: I think these extra imports are left over from some work in progress you were doing. These two imports (from `@fortawesome/...`) aren't actually used being used in this file, so they should be removed.

**Review by travigd — COMMENTED** [review]
Looks mostly pretty good!!!!, but I'm not sure why weird stuff happened to the `package.json` file.

Can you address the one change about the imports and then I'll just touch up the package.json thing for ya?

Thank you!!!!!!!

File: src/components/layout/SiteLayout.tsx (hunk 1 above)
Comment (techniic0l0r) [reply src/components/layout/SiteLayout.tsx:4]: Done! Do I need to create another pull request for the removed lines? I just staged a commit for them. 

I'm not sure what's going on with my files. When I cloned the repo for the conference site, I ended up with an extra copy of conference-site-main alongside conference-site...

Comment (travigd) [reply src/components/layout/SiteLayout.tsx:4]: You can just push your existing commit to the branch. So you should:
* Make sure you're on this branch (`adding-buttons`)
* Push

Comment (techniic0l0r) [reply src/components/layout/SiteLayout.tsx]: Got it, thank you! Hopefully I did that right

**Comment by cmytelka** [comment]
Nice! I would suggest maybe centering the social media icons in the footer, but this might also be a personal preference

**Review by cmytelka — APPROVED** [review]

**Pull request merged**